app/controllers/concerns/wp_preview_tools.rb

```diff
@@ -0,0 +1,19 @@
+#
+# This concern may be mixed into a controller to add common functinality
+# for preview rendering, and preview token validation.
+#
+module WpPreviewTools
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def self.validate_preview_token(wp_post_model)
+      return wp_post_model.status == 'publish'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.